django_stubs_ext/requirements.txt

```diff
@@ -0,0 +1,2 @@
+django
```

Comment:
Do we need this file?

Reply:
it's technically true from an end-user perspective, the package does rely on django, but if someone installs it, they'll probably have django installed anyway

Reply:
ahhhh missed the point of requirements.txt. nevermind, we can remove it